### ISSUE TITLE
Plan for jekyll-multiple-languages-plugin replacement or fork?

### ISSUE THREAD

**Issue by fmichea** [george-gca/multi-language-al-folio#17]
### Have you checked that your feature request isn't already filed?

- [X] I read through [FAQ](https://github.com/george-gca/multi-language-al-folio/blob/master/FAQ.md) and searched through the past issues both in the [original repo](https://github.com/alshedivat/al-folio/issues) and [this one](https://github.com/george-gca/multi-language-al-folio/issues), none of which addressed my issue.
- [X] Yes, I have checked that this feature request isn't already filed.

### Description & Motivation

Hello,

Thanks for making this, I was playing with the repository and particularly the options directly affecting [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin) and ran into their issue https://github.com/kurtsson/jekyll-multiple-languages-plugin/issues/209 which led me to notice that the repository was archived on Jan 15th. The README also specifies that the project was looking for maintainers. This leads me to believe this project might be abandoned, probably something to confirm.

I am creating this issue to discuss the next steps from your point of view. On my end until there is a clear path, I've been thinking at potentially migrating to [untra/polyglot](https://github.com/untra/polyglot), which seems active and maintained with very similar features. What do you think is the best path forward?

### Pitch

Options I can think of:
- Do nothing and keep using [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin) with no change, hoping nothing major breaks. Note that I've already had trouble with changing the ``default_locale_in_subfolder`` option with two different issues.
- Create a maintained fork of the originally used plugin: [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin)
- Move towards [untra/polyglot](https://github.com/untra/polyglot) instead which seems to be active.

**Comment by george-gca** [comment]
Earlier this month I got aware of this situation. In the past [I tried to reach out](https://github.com/kurtsson/jekyll-multiple-languages-plugin/pull/203) with [jekyll-multiple-languages-plugin](https://github.com/kurtsson/jekyll-multiple-languages-plugin/) devs to volunteer being a maintainer. I have already started investigating how much work will be to migrate to polyglot, or even if I like its solution.

For example, in their documentation they state that you should keep the files names consistent, but I don't know if it is possible to keep the different files languages in different directories, which is the current solution for this template and I tend to like most. Imagine if one would keep a blog in 3 different languages, with just a few posts the `_posts` dir would be too crowded.

I hope this weekend I have enough time to look it through and decide what would be the best path.

**Comment by fmichea** [comment]
Thanks for the quick answer, of course I am not expecting a urgent fix, just wanted to start the conversation. The solution with least resistance would likely be to make a fork and migrate this project to the fork, but that does require maintaining it long term.

Just gathering some links and thoughts on polyglot otherwise:
- Sample site: https://polyglot.untra.io/
- Code: https://github.com/untra/polyglot/tree/master/site
- Based on this it looks like the options are pretty open in terms of sub-folder vs flat structure. Does seem to provide a nice way to [internationalize permalinks](https://github.com/untra/polyglot?tab=readme-ov-file#using-different-permalinks-per-language) as well since late October last year.

I'll let you know if I end up experimenting with it!

**Comment by george-gca** [comment]
Update on this issue: I was able to start migration from `jekyll-multiple-languages-plugin` to `polyglot`, and after a bit of a struggle I was able to migrate 90% of the code. There are still a couple of issues that I need to fix, but I hope I can finish the migration this weekend.

When I finish this I will also create a new section for how to update your site if you created yours from the previous version.
